src/exampleTypes/ResultTable.js

```diff
@@ -41,6 +41,84 @@ let popoverStyle = {
   userSelect: 'none',
 }
 
+let HighlightedColumn = withStateLens({ viewModal: false })(
+  observer(
+    ({
+      visibleFields,
+      node,
+      highlight = _.result('slice.0.highlight', getResults(node)),
+      header,
+      Cell = header ? 'th' : 'td',
+      Table = 'table',
+      Modal = null,
+      viewModal,
+    }) => {
+      let alsoHighlighted = _.difference(_.keys(highlight), visibleFields)
+      let getConnector = i => {
```

Comment:
we use `toSentence` for this usually which is available on `underscore.string.fp`

Reply:
https://github.com/stoeffel/underscore.string.fp#tosentencelastdelimiter-delimiter-array--string

If it doesn't support tree shaking (e.g. just picking that one method), we could add it to futil since I think it's the only method not in lodash already 🤔